yt_dlp/extractor/steam.py

```diff
@@ -127,3 +127,44 @@ def _real_extract(self, url):
             raise ExtractorError('Could not find any videos')
 
         return self.playlist_result(entries, playlist_id, playlist_title)
+
+
+class SteamCommunityBroadcastIE(InfoExtractor):
+    _VALID_URL = r'https?://steamcommunity\.(?:com)/broadcast/watch/(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://steamcommunity.com/broadcast/watch/76561199037072858',
+        'only_matching': True,
+    }]
```

Comment:
No proper test?

Reply:
Doesn't this expected test for live stream? If the url ends, it will return `HTTP 410 Gone`

Reply:
rule of thumb for livestreams is to have the fields still be set and then test against those, but set the `skip` field be set to something like `Stream has ended`